core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -417,6 +417,7 @@
       "The amount of time to sleep between checking for the Montior ZooKeeper lock"),
   MONITOR_LOG_DATE_FORMAT("monitor.log.date.format", "yyyy/MM/dd HH:mm:ss,SSS", PropertyType.STRING, "The SimpleDateFormat string used to configure "
       + "the date shown on the 'Recent Logs' monitor page"),
+  MONITOR_HEADER_IMPORTS("monitor.header.imports", "", PropertyType.STRING, "JSON list of CSS and JS files to be imported."),
```

Comment:
I think "resources" might be a better name than "imports".

Reply:
Could also be "monitor.resources.external" to match the class path for the ones we bundle.

Reply:
Sounds good.